core/src/test/java/io/undertow/server/handlers/HeadBlockingExchangeTestCase.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014 Red Hat, Inc., and individual contributors
```

Comment:
thanks for your PR, @carterkozak !
Can you please change the Copyright year to 2022?

Reply:
Updated! Thanks for the review